Add surface rocks to surface ore veins

swagXDragonSlayer46YT opened the pull request:
Added surface rocks for:
- Cassiterite
- Smithsonite
- Salt flats

swagXDragonSlayer46YT committed:
Add surface rocks to surface ore veins

Added surface rocks for:
- Cassiterite
- Smithsonite
- Salt flats

bruberu commented:
Personally, I think we should have surface rocks for all overworld veins, given that most of these are needed before MV, when one actually gets to the prospector.

swagXDragonSlayer46YT committed:
More ore vein changes

Made it so that all overworld ore veins go from y = 0 to y = 80. All overworld ore veins also have surface rocks.
